### ISSUE TITLE
submit token idea

### ISSUE THREAD

**Issue by Blockjock** [FreeLoaderz/rwd-frontend#30]
once I sign transaction what if the token claim field slowly de-materializes like the transporter on Star Trek? I know its probably not easy, I am just thinking out loud as to what would be cool.

**Comment by Sbcdn** [comment]
Postponed for later if we have time

**Comment by fisherstevenk** [comment]
Again, this would have to be part of a theme for the website. 

I'm down for whatever the majority wants to do.  I'm personally more a "clean" design type person vs some of these other "busy" websites

**Comment by Blockjock** [comment]
so in webdev they really dont have anything already written for this? if the UI has some differences in design it could get peeps to talk about it more, before cup holders in cars everyone thought why would you want that? then someone put one in there and boom every car had one. It gets peeps to talk about the platform, because lets face it, were just dropping tokens into peoples wallets, that can get pretty boring after a bit.

**Comment by fisherstevenk** [comment]
They do.  

Like I mentioned though, we'll want a theme.  Wouldn't do "water" and the "dematerialize" on the same site.  Know what I mean?

So we'd have to choose one. 

I'd lean towards the second one.

**Comment by Blockjock** [comment]
yep agreed think the second one is cool but dematerializing is something no one is doing. the idea of the space cowboy (torben loved) would go along weith the de,aterialized feature very well.

**Comment by fisherstevenk** [comment]
There are "angular animations" that I tried playing with tonight a little bit.. it didn't quite work the way I was thinking.  I'll just have to mess with it more.
